test/factories/site.rb

```diff
@@ -0,0 +1,6 @@
+FactoryGirl.define do
+  factory :site do
+    name "daimon-news"
+    fqdn "www.example.com"
```

Comment:
`Site#fqdn` should be a uniqueness attribute.
So should we make it also uniqueness in test data?